### PR TITLE
Modify style.css footer.css and base.html

### PR DESCRIPTION
**Pull request by anuschka**
Implemented sticky footer and added padding and margins for the main content. Here working on #45

### CODE REVIEW

File: static/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* Sticky footer styles
+		-------------------------------------------------- */
+
```
Comment (ialja) [inline static/css/footer.css]: I'd suggest removing the 3 tab indentation in this file.

File: templates/base.html [hunk 2]
```diff
@@ -8,8 +8,13 @@
 		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
 		<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<link rel="stylesheet" href="{% static 'css/style.css' %}">
+		<link rel="stylesheet" href="{% static 'css/footer.css' %}">
```
Comment (ercchy) [inline templates/base.html]: @anuschka no need for additional .css file. Everything can be in one having multiple files is not optimal as the client makes request for everyone separately.

Comment (anuschka) [inline templates/base.html]: OK. Will move all to style.css.

**Comment by anuschka** [comment]
Thanks :-)

**Pull request merged**